src/api/exceptions/InvalidParamException.ts

```diff
@@ -3,7 +3,7 @@
 // file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE
 
 /**
- * MessageException
+ * InvalidParamException
  * ----------------------------------------
  *
  * This should be used if a someone requests a
```

Comment:
This should be used if the command that was called was called with invalid parameter...